fab/services/templates/supervisor_formsplayer_spring.conf

```diff
@@ -0,0 +1,8 @@
+[program:{{ project }}-{{ environment }}-formsplayer-spring]
+command=java -jar {{ code_current}}/submodules/formsplayer/build/libs/formsplayer.jar
+user={{ sudo_user }}
+autostart=true
+autorestart=true
+stdout_logfile={{ log_dir }}/formsplayer-spring.log
+redirect_stderr=true
+stderr_logfile={{ log_dir }}/formsplayer-spring.error.log
```

Comment:
FWIW a Jenkins build would be pretty easy

On Wed, Jan 20, 2016 at 10:31 AM, Ben <notifications@github.com> wrote:

> In fab/services/templates/supervisor_formsplayer_spring.conf
> <https://github.com/dimagi/commcare-hq/pull/9947#discussion_r50268818>:
>
> > @@ -0,0 +1,8 @@
> > +[program:{{ project }}-{{ environment }}-formsplayer-spring]
> > +command=java -jar {{ code_current}}/submodules/formsplayer/build/libs/formsplayer.jar
> > +user={{ sudo_user }}
> > +autostart=true
> > +autorestart=true
> > +stdout_logfile={{ log_dir }}/formsplayer-spring.log
> > +redirect_stderr=true
> > +stderr_logfile={{ log_dir }}/formsplayer-spring.error.log
>
> [image: :+1:] for startsecs. i think the .jar itself sets the port? or at
> least the default one. may be good to explicitly set it though so when
> people read this they know where the entry point is
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9947/files#r50268818>.
>